CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [v1.0.0] - 2023-08-25
 
 ### Added
+- Add support for `--literal` from [PanGan21](https://github.com/PanGan21)
```

Comment:
Should this have gone to an already-released v1.0.0?

Reply:
good catch! actually, I was planning to use the GitHub auto-generate release note, this file may not be necessary, can you help create a PR to fix it and add a notice that we are no longer using this file, we use the GitHub auth-generate one instead.

Reply:
Not exactly sure what you meant, but opened one here: https://github.com/lsd-rs/lsd/pull/907

Let me know if it is what you meant.